### ISSUE TITLE
Create a release

### ISSUE THREAD

**Issue by m-kuhn** [linuxdeploy/linuxdeploy-plugin-qt#139]
At the moment, there is a github action that releases latest `master` with tag `continuous`.
We use the assets of this in an action.
Because of [an issue with current master](https://github.com/linuxdeploy/linuxdeploy-plugin-qt/pull/136) currently our actions fail. It would be nice if one (or more) fixed tags would be offered to be able to develop against something stable.

**Comment by TheAssassin** [comment]
Good point. Once the ARM builds will be working, there will be tags similar to the ones in linuxdeploy.
